.packit.yaml

```diff
@@ -0,0 +1,39 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: rubygem-foreman_maintain.spec
+
+# add or remove files that should be synced
+files_to_sync:
+    - rubygem-foreman_maintain.spec
+    - foreman_maintain.logrotate
+    - .packit.yaml
+
+# name in upstream package repository or registry (e.g. in PyPI)
+upstream_package_name: foreman_maintain
+# downstream (Fedora) RPM package name
+downstream_package_name: rubygem-foreman_maintain
+
+upstream_tag_template: v{version}
+
+actions:
+  post-upstream-clone:
+    - "wget https://raw.githubusercontent.com/theforeman/foreman-packaging/rpm/develop/packages/foreman/rubygem-foreman_maintain/rubygem-foreman_maintain.spec -O rubygem-foreman_maintain.spec"
+    - "wget https://raw.githubusercontent.com/theforeman/foreman-packaging/rpm/develop/packages/foreman/rubygem-foreman_maintain/foreman_maintain.logrotate -O foreman_maintain.logrotate"
+  create-archive:
+    - gem build foreman_maintain.gemspec
+    - bash -c "ls -1t ./foreman_maintain-*.gem | head -n 1"
+
+jobs:
+  - job: copr_build
+    trigger: pull_request
+    targets:
+      centos-stream-8:
+        additional_modules: "foreman:el8,nodejs:12"
+        additional_repos:
+          - http://koji.katello.org/releases/yum/foreman-nightly/el8/x86_64/
+    module_hotfixes: true
```

Comment:
this is required to make dnf install maintain from this repo instead of ours, as we all know ;)